javascript/exoservice/bin/cuc-api

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+set -ex
```

Comment:
I'm not sure this is adding that much value either. Do you really want to see debug output for all these `if` clauses? All it does is call cucumber-js.